docs/deployment_patterns/how_to_use_great_expectations_in_flyte.md

```diff
@@ -0,0 +1,202 @@
+---
+title: How to Use Great Expectations in Flyte
+---
+import Prerequisites from '../guides/connecting_to_your_data/components/prerequisites.jsx'
+
+*This piece of documentation was authored by [Samhita Alla](https://www.linkedin.com/in/samhita-alla) and the [Flyte](https://flyte.org/) team*
+
+This guide will help you run a Great Expectations in [Flyte](https://flyte.org/)
+
+<Prerequisites>
+
+- [Set up a working deployment of Great Expectations](../tutorials/getting_started/intro.md)
+- [Created an Expectation Suite](../tutorials/getting_started/create_your_first_expectations.md)
```

Comment:
The intro walks users through the entire experience from initializing a data context, through connecting to data, through creating an expectation suite. If we're going to have a specific step for "Creating an Expectation Suite" (which make sense), I think we should have a step right before for ["Connecting to Data"](https://docs.greatexpectations.io/docs/tutorials/getting_started/connect_to_data).

I just want to call out that we are linking to the Getting Started tutorial here, which is about specifically working with the taxi data. I don't think we have other good generic links for this, so this is probably ok - for instance, our links about Connecting to Data would otherwise all be particular to specific backends.